codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -1170,5 +1170,21 @@ public final V setValue(V value) {
         public final String toString() {
             return key.toString() + '=' + value.toString();
         }
+
+        @Override
+        public boolean equals(Object o) {
+            if (!(o instanceof Map.Entry)) {
+                return false;
+            }
+            @SuppressWarnings("rawtypes")
+            Map.Entry other = (Map.Entry) o;
```

Comment:
If you can, it's still be better to cast to `Map.Entry<?, ?>`. Removing the generics can lead to surprising cases (none here, but in general strongly avoid rawtypes).

Reply:
@ryu1-sakai please do what @ejona86 suggested and ping me once done :)